stable/nginx-ingress/README.md

```diff
@@ -49,7 +49,8 @@ Parameter | Description | Default
 `controller.image.repository` | controller container image repository | `quay.io/kubernetes-ingress-controller/nginx-ingress-controller`
 `controller.image.tag` | controller container image tag | `0.18.0`
 `controller.image.pullPolicy` | controller container image pull policy | `IfNotPresent`
-`controller.config` | nginx ConfigMap entries | none
+`controller.configMap.name` | point to an existing configMap rather than creating one | ""
```

Comment:
`controller.configMap` | point to an existing configMap rather than creating one | ~